site/content/docs/4.3/examples/cover/index.html

```diff
@@ -23,13 +23,11 @@ <h3 class="masthead-brand">Cover</h3>
     <h1 class="cover-heading">Cover your page.</h1>
     <p class="lead">Cover is a one-page template for building simple and beautiful home pages. Download, edit the text, and add your own fullscreen background photo to make it your own.</p>
     <p class="lead">
-      <a href="#" class="btn btn-lg btn-secondary">Learn more</a>
+      <a href="#" class="btn btn-lg btn-secondary border-white">Learn more</a>
     </p>
   </main>
 
   <footer class="mastfoot mt-auto">
-    <div class="inner">
-      <p>Cover template for <a href="https://getbootstrap.com/">Bootstrap</a>, by <a href="https://twitter.com/mdo">@mdo</a>.</p>
-    </div>
+    <p>Cover template for <a href="https://getbootstrap.com/">Bootstrap</a>, by <a href="https://twitter.com/mdo">@mdo</a>.</p>
```

Comment:
It seems there are a few instances of `inner` but I don't see anything in the CSS. Maybe remove those too?

Reply:
good idea. I've updated the PR.

Reply:
BTW it seems the whole cover example could be simplified a lot more by using more utils.

Let's wait for the others to chime in, but it might be worth tackling all of it here.

Reply:
Cool. One thing I was wondering about. It'd be good if the examples could specify in the frontmatter a CSS class to add to the `<html>` tag. For example for this one we could add:
`html_class: "h-100"` so that would add the `h-100` class to the `html` tag. Then this could be removed from the CSS. I think there's a few examples that could take advantage of specifying CSS classes in the `html` tag like this.

Reply:
I'm definitely in favor of `html_class: "h-100"`!

Reply:
This is already possible for the examples. https://github.com/twbs/bootstrap/blob/master/site/layouts/_default/examples.html#L2